### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rfsv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # Servo's rust-bindgen
 
-A binding generator for the Rust language.
-
+A binding generator for the Rust language.</br>
```
Comment (emilio) [inline README.md]: Please use actual newlines instead.

File: README.md [hunk 2]
```diff
@@ -57,10 +54,10 @@ source. For that, follow the instructions
 
 Those instructions list optional steps. For bindgen:
 
-* Checkout and build clang
-* Checkout and build the extra-clang-tools
-* Checkout and build the compiler-rt
-* You do not need to checkout or build libcxx
+* Checkout and build clang;
```
Comment (emilio) [inline README.md]: Why a semi-colon and not a colon?

File: README.md [hunk 3]
```diff
@@ -84,8 +81,7 @@ scripts][stylo-scripts] to see how is it used inside the Servo organisation.
 
 ## C++ Usage
 
-This fork of rust-bindgen can handle a number of C++ features.
-
+This fork of rust-bindgen can handle a number of C++ features.</br>
```
Comment (emilio) [inline README.md]: ditto re. newlines

File: README.md [hunk 4]
```diff
@@ -148,5 +144,5 @@ and `Clone` traits for a type.
 This mode isn't actively maintained, so no promises are made around it. Check
 out the upstream documentation for info about how it *should* work.
 
-[sm-script]: https://github.com/servo/rust-mozjs/blob/master/etc/bindings.sh
-[stylo-scripts]: https://github.com/servo/servo/tree/master/ports/geckolib/gecko_bindings/tools
+[sm-script]: https://github.com/servo/rust-mozjs/blob/master/etc/bindings.sh;
```
Comment (emilio) [inline README.md]: These are links and shouldn't have any colon/semi-colon.

**Comment by KiChjang** [comment]
Closing since this looks to be a spam account.

**Pull request closed without merging**